dev/InfoBar/InfoBarPanel.cpp

```diff
@@ -27,11 +27,11 @@ winrt::Size InfoBarPanel::MeasureOverride(winrt::Size const& availableSize)
         if (childDesiredSize.Width != 0 && childDesiredSize.Height != 0)
         {
             // Add up the width of all items if they were laid out horizontally
-            const auto horizontalMargin = winrt::InfoBarPanel::GetHorizontalMargin(child);
+            const auto horizontalMargin = winrt::InfoBarPanel::GetSpacingInHorizontalOrientation(child);
             totalWidth += childDesiredSize.Width + (nItems > 0 ? (float)horizontalMargin.Left : 0) + (float)horizontalMargin.Right;
```

Comment:
totalWidth is ignoring the left margin on the first item, shouldn't it also ignore the right margin on the last item?

Reply:
I think it should, but I am not sure why it is not. @teaP do you know why right margin is not being ignored on the last item?

Reply:
I think it should, but it currently does not seem to affect anything since the last item's horizontal right margin value is 0.

Reply:
Tracking this in a separate issue #3713 since this is not related to the API name change.